stellar-multitenant/pkg/internal/validators/tenant_validator.go

```diff
@@ -104,12 +104,22 @@ func (tv *TenantValidator) ValidateUpdateTenantRequest(reqBody *UpdateTenantRequ
 		if _, err = url.ParseRequestURI(*reqBody.BaseURL); err != nil {
 			tv.Check(false, "base_url", "invalid base URL value")
 		}
+
+		*reqBody.BaseURL, err = utils.SetURLSchemeIfEmpty(*reqBody.BaseURL)
+		if err != nil {
+			tv.Check(false, "base_url", "invalid base URL value. Verify the URL protocol scheme")
```

Comment:
You're getting rid of the error here. Can you at least log the error since it's not being added to the check message?